C++/include/algorithm/dynamic_programming/README.md

```diff
@@ -44,3 +50,196 @@ Time               | Space
 _O(N<sup>3</sup>)_ | _O(N<sup>2</sup>)_
 
 where N is the number of matrices.
+
+## 2. Coin Change
+
+Given an unlimited amount of coins with different values, find the number of ways of making change for an given
```

Comment:
Rather than putting in your own newlines (hard-wrapping), put each paragraph on one line. Github (and IDEs) will manage the soft-wrapping for you. Please update the rest of your paragraphs, too.

Reply:
…making change for **a** given…